src/metabase/models/audit_log.clj

```diff
@@ -64,12 +64,15 @@
   `object` can also be a map of arbitrary details relavent to the event, which is recorded as-is. If the name and/or ID
   of a model are also relevant to the event and should be recorded, they can be passed as third and fourth arguments."
```

Comment:
I think the docstring here should read as :

`If the name and/or ID of a model are also relevant to the event and should be recorded, they can be passed as  fourth and fifth arguments.`

Because I think the third arg is `user-id`.

Reply:
Thanks, yeah I changed the function signature but forgot to update the doc string